### ISSUE TITLE
Chart Specs default loaded from same asset host as the cedar JS

### ISSUE THREAD

**Issue by ajturner** [Esri/cedar#101]
```javascript
<script src="https://myhost/js/cedar.js"></script>
<script>
new Cedar({
  "type": "bar"
});
```

would load `https://myhost/js/specs/bar.json`

```javascript
<script src="https://myhost/js/cedar.js"></script>
<script>
new Cedar({
  "type": "http://gist.github.com/ajturner/bar-awesome.json"
});
```

would load `http://gist.github.com/ajturner/bar-awesome.json`

**Comment by tomwayson** [comment]
@ajturner 

I was planning on using the specs under [src/charts](https://github.com/Esri/cedar/tree/master/src/charts), but it looks like the those are out of date when compared with the ones under [site/source/data/templates](https://github.com/Esri/cedar/tree/master/site/source/data/templates). So I plan to copy those over to the src/charts folder, but want to verify w/ you which ones. Seems like all but:

scatter-template-v1.json - this should prob be deleted.
scatter-complex.json
bar-complex.json
time-dots.json?

Basically, which ones ship "in the box" and which are just part of the site examples?

Also plan to rename horizontal to bar-horizontal.json.

**Comment by tomwayson** [comment]
I meant to say I will *move* the specs from the site templates folder into the src/charts folder, and then update the site accordingly.

**Comment by ajturner** [comment]
Correct, I think just move these to src/charts

* bar
* bar-horizontal
* scatter
* bubble
* pie
* time

Then we can add more supported charts there (e.g. histogram, line)
